client/src/node/extension.ts

```diff
@@ -101,15 +106,24 @@ export function activate(context: ExtensionContext): void {
   // If configFile setting is changed, update watcher to watch new configuration file
   workspace.onDidChangeConfiguration((event: ConfigurationChangeEvent) => {
     if (event.affectsConfiguration("SAS.connectionProfiles")) {
-      const profileList = profileConfig.getAllProfiles();
-      const activeProfileName = profileConfig.getActiveProfile();
-      if (activeProfileName in profileList || activeProfileName === "") {
-        updateStatusBarProfile(activeProfileStatusBarIcon);
-      } else {
-        profileConfig.updateActiveProfileSetting("");
-      }
+      triggerProfileUpdate();
     }
   });
+
+  triggerProfileUpdate();
+}
+
+function triggerProfileUpdate(): void {
+  const profileList = profileConfig.getAllProfiles();
+  const activeProfileName = profileConfig.getActiveProfile();
+  if (activeProfileName in profileList || activeProfileName === "") {
+    updateStatusBarProfile(activeProfileStatusBarIcon);
+    // TODO Update with changes for SAS 9 integration
+    // commands.executeCommand("setContext", "SAS.connectionMethod", profileList[activeProfileName].connectionMethod || "rest");
+    commands.executeCommand("setContext", "SAS.connectionMethod", "rest");
```

Comment:
@smorrisj I put something in place here that we can hopefully tweak w/ SAS 9 integration.

Reply:
@scottdover still working out the connection type values that are accepable. In my working model I have "viya" and "ssh" atm.